Agreements/Airport/BB_Stafa.txt

```diff
@@ -1,2 +1 @@
-;FixMe LUVUM/ADMEX etc?
-COPX:EGBB:*:WHI:*:*:BBAPP:Stafa:8000:*:^WHI
\ No newline at end of file
+COPX:EGBB:*:LUVUM:*:BBAPP:Stafa:8000:*:^LUVUM
```

Comment:
```suggestion
COPX:EGBB:*:LUVUM:*:*:BBAPP:Stafa:8000:*:^LUVUM
COPX:EGBB:*:*:TNT:*:BBAPP:Stafa:8000:*:^TNT
COPX:EGBB:*:*:NANTI:*:BBAPP:Stafa:8000:*:^NANTI
```

Reply:
Have added TNT and NANTI COPX as back-up, in case the aircraft routes north of LUVUM or for some weird reason is on the TNT non-radar SIDs. Wouldn't put it past anyone...